src/extra/imgui.cpp

```diff
@@ -947,8 +1041,14 @@ struct Text : public Base {
         _label = value.payload.stringValue;
       }
     } break;
-    case 1:
-      _color = value;
+    case 1: {
+      if (value.valueType == None) {
+        _variable_name.clear();
+      } else {
+        _variable_name = value.payload.stringValue;
+      }
+      cleanupVariable();
```

Comment:
Can't rem why I did this really. Anyway ParamVar would be ideal as mentioned above